scripts/run_portserver.py

```diff
@@ -420,7 +422,11 @@ class Server:
     max_backlog = 5
     message_size = 1024
 
-    def __init__(self, handler: Callable[..., Any], socket_path: str) -> None:
+    def __init__(
+        self,
+        handler: Callable[[bytes], Optional[Union[str, bytes]]],
```

Comment:
Can we pick either str or bytes for the second item here, and be consistent throughout? Or, for that matter, int?

Reply:
following the documentation that was provided by chris (https://github.com/oppia/oppia/pull/16226#discussion_r996248702 ) I think here we should allow bytes.